requirements/default.txt

```diff
@@ -1,4 +1,3 @@
-numpy>=1.10
+numpy>=1.13
```

Comment:
cartopy only support python 3.5+, and  numpy only started 3.5 support in v1.13.3

Reply:
We are actually only testing against 3.6 on master now, so it is even newer than that!

Reply:
probably worth doing NEP29 and pyupgrade in a follow-up PR